keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ErrorSimulationControllerTests.kt

```diff
@@ -3,23 +3,23 @@ package com.netflix.spinnaker.keel.rest
 import com.netflix.spinnaker.keel.KeelApplication
 import com.netflix.spinnaker.keel.spring.test.MockEurekaConfiguration
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
 import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.autoconfigure.task.TaskSchedulingAutoConfiguration
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
 import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.test.context.junit.jupiter.SpringExtension
 import org.springframework.test.web.servlet.MockMvc
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
 
-@ExtendWith(SpringExtension::class)
 @SpringBootTest(
   classes = [KeelApplication::class, MockEurekaConfiguration::class],
   properties = ["tests.error-simulation=true"],
   webEnvironment = SpringBootTest.WebEnvironment.MOCK
 )
 
 @AutoConfigureMockMvc
+@EnableAutoConfiguration(exclude = [TaskSchedulingAutoConfiguration::class])
```

Comment:
I wonder if we should make a `DefaultTestConfiguration` class with these common annotations for Spring tests...

Reply:
Yeah, I was considering making an annotation that groups some things.